go/mysql/flavor.go

```diff
@@ -132,6 +133,11 @@ type flavor interface {
 	// until the context expires. It returns an error if we did not
 	// succeed.
 	waitUntilPosition(ctx context.Context, c *Conn, pos replication.Position) error
+	// catchupToGTIDCommands returns the command to catch up to a given GTID.
+	catchupToGTIDCommands(params *ConnParams, pos replication.Position) []string
+
+	// binlogReplicaField returns the field to use to check replica updates.
+	binlogReplicaField() string
```

Comment:
IMO a better name would be `binlogReplicatedUpdates`.  From the current name, I had no idea what it was actually checking.

Reply:
Fixing